src/ansys_sphinx_theme/theme/ansys_sphinx_theme/components/breadcrumbs.html

```diff
@@ -1,30 +1,45 @@
 {% block breadcrumbs %}
 <nav aria-label="{{ _('Breadcrumb') }}">
   <ul class="bd-breadcrumbs">
-    {# Additional breadcrumbs before "Home" #} {% block rootrellink %} {% for
-    (name, href) in theme_additional_breadcrumbs %}
+    {# Additional breadcrumbs before "Home" #} {%- if
+    theme_additional_breadcrumbs %} {% block rootrellink %} {# Home icon for
+    only one theme_additional_breadcrumbs #} {% for name, href in
+    theme_additional_breadcrumbs %} {% if loop.first %} {# Home icon #}
     <li class="breadcrumb-item breadcrumb-home">
       <a href="{{ href }}" class="nav-link">{{ name }}</a>
     </li>
-    {% endfor %} {% endblock %} {# Home icon #}
+    {% else %} {# Breadcrumb item #}
     <li class="breadcrumb-item">
+      <a href="{{ href }}" class="nav-link">{{ name }}</a>
+    </li>
+    {% endif %} {% endfor %}
+    <li class="breadcrumb-item">
+      <a
+        href="{{ pathto(root_doc) }}"
+        class="nav-link"
+        aria-label="{{ _('Home') }}"
+      >
+        <i class="fa-solid fa-home"></i>
+      </a>
+    </li>
+    {% endblock %} {%- else %} {# Home icon #}
+    <li class="breadcrumb-item breadcrumb-home">
```

Comment:
I would have expected that line 26 (``breadcrumb-item breadcrumb-home``) would go earlier in the file...like before the line 13 (before the loop) because the house icon should go at the beginning of the breadcrumps?

Not sure if makes sense my thinking....

Reply:
House icon refers to the "home" location for your docs. Makes sense that it goes down the road that much.